hs/t/n/API_OutBeforeIn.txt

```diff
@@ -3,11 +3,11 @@ Verifying for generic connector
   Verifying when ALL participants are honest
   Verifying when NO participants are honest
 Checked 4 theorems; No failures!
-reachc: error[RAPI0001]: API "Writer_f" calls interact.out() before interact.in()
+reachc: error[RAPI0000]: API "Writer_f" does not occur in program
```

Comment:
This seems like a weird error? Is it because EPP didn't see the `in` and so it didn't separate it out?